content/blog/page-view-counter/index.md

```diff
@@ -0,0 +1,273 @@
+
+The simple way to store entities is to create a class that inherits from `TableEntity`. What this means is that the class will need to have two Properties: A `PartitionKey` and a `RowKey` . Both keys, combined, form the **Key** through which we will **uniquely identify our entity**. However, separated they serve a different purpose:
+
+-  The `PartitionKey` defines the partition in which the service will store the entity: 
+>Tables are partitioned to support load balancing across storage nodes. A table's entities are organized by partition. A partition is a consecutive range of entities possessing the same partition key value. The partition key is a unique identifier for the partition within a given table, specified by the PartitionKey property. 
+
+-  The `RowKey` is a unique identifier for an entity within a given partition.
+
+  Since the `PartitionKey` is used for load balancing, it's important that it's not hardcoded. For this, we can use the URL of the page in which we are counting the views. The `RowKey` can be hardcoded, because we are already changing the key through the use of the URL in the `PartitionKey`. Eventually we could have different `RowKey` for different scenarios, but for now it will stay as a hardcoded value. You can read more [here](https://docs.microsoft.com/en-us/rest/api/storageservices/understanding-the-table-service-data-model).
+
+  After that, the entity can have whatever fields we want. Since we are storing the values, this is how our class will look:
+```csharp
+  public class ViewCount : TableEntity
+    {
+        public ViewCount(string URL)
+        {
+            this.PartitionKey = URL; //the partition key for load-balancing and identification
+            this.RowKey = "visits"; // hardcoded for identification
+            Count = 0; // the value we want to store
+        }
+
+        public int Count { get; set; }
+
+        public ViewCount()
+        {
+            Count = 0;
+        }
+    }
+```
+### Changing our function to connect to Storage
+
+We now have the entity that we want to store and retrieve, so we need to connect to Storage and do precisely that. Table entities are stored in Tables which are identified with a name, so we can choose it and connect to it through a `CloudTable` object. In general, the logic is as follows:
+
+1. Connect to the Storage Account through the connection string.
+1. Create a table client through which we do transactions (inserts, updates, etc.).
+1. Get the table through its name. 
+1. Execute transactions on the table.
+
+Here are the first 3 steps in code:
+
+```csharp
+    const string tableName = "viewcountertable";
+    
+    var storageAccount = CloudStorageAccount.Parse($"{yourStorageAccountConnectionString}");
+    var tableClient = storageAccount.CreateCloudTableClient();
+    CloudTable table = tableClient.GetTableReference(tableName);
+    
+    await table.CreateIfNotExistsAsync(); // we can let our code create the table if needed
+```
```

Comment:
I would love to see which files are these

Reply:
Alternatively you can simply do the famous `putting it all together` section where you show the entire code working together